public/wordHints.js

```diff
@@ -88,123 +88,28 @@ const wordHints = {
-  // },
-  // COOL: {
-  //   hints: [`Not warm`, "Not cold"],
-  // },
-  // CROW: {
-  //   hints: [`A kind of bird`, `It's black`],
-  // },
-  // CARD: {
-  //   hints: [`Made of paper`, `Used in many games`],
-  // },
-  // CAME: {
-  //   hints: ["Moved closer", 'Opposite of "went away"'],
-  // },
-  // COAT: {
-  //   hints: [`Something you wear`, "🧥"],
-  // },
-  // CANE: {
-  //   hints: [`Candy ____`, "🦯"],
-  // },
-  // CORN: {
-  //   hints: ["Something to eat", "🌽"],
-  // },
-  // CONE: {
-  //   hints: ["A shape", "Eaten with ice cream"],
-  // },
-  // CART: {
-  //   hints: ["Something you push", "🛒"],
-  // },
-  // COOK: {
-  //   hints: ["Something you do to food", "🍳"],
-  // },
-  // CUBE: {
-  //   hints: [`A shape`, "🧊"],
-  // },
-  // COWS: {
-  //   hints: ["Tame animals", "🐮"],
-  // },
-  // CASE: {
-  //   hints: [`A container`, `A pillow ____`],
-  // },
-  // CREW: {
-  //   hints: [`Group of workers`, `People working on a boat`],
-  // },
-  // CRAM: {
-  //   hints: [`Put a lot of things inside`],
-  // },
-  // CLAP: {
-  //   hints: ["Hand noise", "👏"],
-  // },
-  // CLUB: {
-  //   hints: [`A golf ____`, "♣️"],
-  // },
-  // CRAB: {
-  //   hints: ["Creature that pinches", "🦀"],
-  // },
-  // COLD: {
-  //   hints: [`Feeling in winter`, "🐧"],
-  // },
-  // CLAY: {
-  //   hints: [`Like mud`, `Used to make pottery`],
-  // },
-  // CURL: {
-  //   hints: [`A twisty shape`, `When hair is not straight`],
-  // },
-  // CAGE: {
-  //   hints: [`Animals could be kept inside`, `Made of metal`],
-  // },
-  // CANS: {
-  //   hints: [`Made of metal`, `Food or drink are kept inside`],
-  // },
-  // CATS: {
-  //   hints: ["A pet", "🐈"],
-  // },
-  // CARS: {
-  //   hints: ["Have wheels", "🚗"],
-  // },
-  // COME: {
-  //   hints: [`Move toward`, `____ on, you can do it!`],
-  // },
-  // CUBS: {
-  //   hints: [`Young creatures`, `Baby bear`],
-  // },
-  // CRIB: {
-  //   hints: [`Small bed`, "Bed for baby"],
-  // },
-  // CHIN: {
-  //   hints: [`On your face`, `Below your mouth`],
-  // },
-  // CAMP: {
-  //   hints: [`Sleeping in nature`, `🏕️`],
-  // },
+  KICK: { category: "Use your foot" },
+  AFTER: { category: "Not before" },
+  EGGS: { emoji: "🥚" },
+  BUNNY: { emoji: "🐰" },
+  MILK: { emoji: "🥛" },
+  TOAD: { category: "Like a frog", emoji: "🐸" },
+  SANK: { category: "Fell to the bottom" },
+  AGREE: { category: "Say yes to" },
+  KEEP: { category: "Hold onto" },
+  EARLY: { category: "Not late" },
+  EARTH: { category: "Third from the Sun", emoji: "🌎" },
+  CAST: { category: "Broken bone cover" },
+  BLINK: { category: "Eye action" },
+  PUMP: { category: "Fill your tire" },
+  GLUE: { category: "Sticky stuff" },
+  SAID: { category: "Spoke" },
+  AGAIN: { category: "Repeat" },
+  WING: { category: "Part of a bird" },
+  BERRY: { category: "A food", emoji: "🍓" },
+  RING: { emoji: "💍" },
+  GRIND: { category: "Crush it" },
+  SONG: { category: "Really hope", emoji: "🎶" },
```

Comment:
I don't get "really hope", but others LGTM

Reply:
oops. copy and paste error